pkg/upstream/testdata/fuse.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
```

Comment:
We usually put test fixtures somewhere in the `hack` directory, so that tests in other packages can refer to the same files without needing to copy them around. When referencing the fixtures, all tests are executed with `CWD` set to the package directory being tested, so you can just use relative paths to get at them